Nuget.config

```diff
@@ -2,7 +2,7 @@
 <configuration>
   <packageSources>
     <clear />
-    <add key="data_api_builder_build_packages" value="https://pkgs.dev.azure.com/sqldab/fcb212b3-b288-4c9e-b55a-5842a268b16d/_packaging/data_api_builder_build_packages/nuget/v3/index.json" />
+    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
```

Comment:
We shouldnt use nuget.org directly, should upstream the necessary packages to our central packages feed

Reply:
ah my bad. That was to unblock me from testing this without going through manually adding all those package upstream. Reverting.

Reply:
reverted.

Reply:
and also updated upstream feed with latest versions of packages.

Microsoft.AspNetCore.Authorization
Microsoft.AspNetCore.Authentication.JwtBearer
Microsoft.AspNetCore.Mvc.Testing
Microsoft.AspNetCore.TestHost
6.0.29
8.0.4

Verify.MsTest
23.6.0

Verify.DiffPlex
2.3.0

ZiggyCreatures.FusionCache
1.1.0
1.0.0

System.IO.Abstractions.TestingHelpers
System.IO.Abstractions
21.0.2

System.IdentityModel.Tokens.Jwt
7.5.1
7.5.0

System.Drawing.Common
8.0.4

MSTest.TestFramework
MSTest.TestAdapter
3.3.1
3.2.2

Microsoft.OpenApi.Readers
Microsoft.OpenApi
1.6.14

Microsoft.OData.Core
Microsoft.OData.Edm
7.21.1
7.20.0

Microsoft.NET.Test.Sdk
17.9.0

Microsoft.IdentityModel.JsonWebTokens
7.5.1
7.5.0

Microsoft.Extensions.Logging.ApplicationInsights
Microsoft.ApplicationInsights.AspNetCore
2.22.0

Microsoft.Extensions.Configuration.Json
Microsoft.Extensions.Configuration.Binder
8.0.0

Microsoft.Data.SqlClient
5.2.0

Microsoft.AspNetCore.Http
2.2.2

DotNetEnv
3.0.0

Azure.Identity
1.11.2